src/signalrclient/url_builder.cpp

```diff
@@ -29,7 +29,7 @@ namespace signalr
 
         void append_connection_token(web::uri_builder &builder, const utility::string_t &connection_token)
         {
-            if (connection_token != _XPLATSTR(""))
+            if (connection_token.length() > 0)

```

Comment:
It just does `return (this->_Mysize == 0);` and I found `.length() > 0` more expressive in case of strings.